app/packages/utilities/src/schema.test.ts

```diff
@@ -0,0 +1,72 @@
+import { describe, expect, it } from "vitest";
+import * as schema from "./schema";
+
+const SCHEMA: schema.Schema = {
```

Comment:
I would suggest re-working these tests to allow for a varied set of schemas, eg. an empty schema.

Reply:
I added empty schema tests. Are there other cases you had in mind?